test/files/neg/t8323.scala

```diff
@@ -0,0 +1,7 @@
+object Test {
+  final val a = "Bippy"
+  final val b = "Dingo"
+  def f(x: a.type) = x
+  def f(x: b.type) = x
+  def main(args: Array[String]): Unit = ()
+}
```

Comment:
Would it be possible, at some point, to be able to define overloads like here? That could be convenient for defining JavaScript facades.